build.gradle

```diff
@@ -14,6 +14,7 @@ repositories {
     maven { url 'http://repo.labs.intellij.net/central-proxy' }
     maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service' }
     mavenCentral()
+    mavenLocal()
```

Comment:
I do not have strict opinion about this. Just reuse artifacts that I have locally.